docs/application-connector/docs/013-details-api.md

```diff
@@ -11,6 +11,11 @@ Find the Application Connector API documentation in the included Swagger files.
 
 For convenient viewing, open these files using [this](https://editor.swagger.io/) Swagger editor.
 
+You can acquire the API specification of the Metadata Service for a given version using the following command:
+```
+curl https://gateway.{CLUSTER_NAME}.kyma.cx/{RE_NAME}/v1/metadataapi.yaml
+```
+
 You can acquire the API specification for a given version directly from the Connector Service:
 ```
 curl https://connector-service.{CLUSTER_NAME}.kyma.cx/v1/api.yaml
```

Comment:
Like previous one, perhaps it is worth to consider adding minikube subsection for fetching API spec